testing/citest/tests/google_http_lb_upsert_scenario.py

```diff
@@ -480,7 +480,17 @@ def update_port_range(self):
   def add_cert(self, certname, title):
     '''Add cert to targetHttpProxy to make it a targetHttpsProxy.
     '''
+    # Note: We add this health check to the upsert description because
+    # this health check would be orphaned otherwise. This allows the 'delete'
+    # test to delete all the resources we allocated.
```

Comment:
I'm still hung up on this.
If before the healthCheck was unchanged, so why would it be orphaned? Now you have a different one, so what happens to the original? If anything I'd think now would lead to something orphaned. Please explain.